pkg/master/master.go

```diff
@@ -189,11 +189,6 @@ func (m *Master) InstallAPIs(c *Config) {
 			ParameterCodec:       api.ParameterCodec,
 			NegotiatedSerializer: api.Codecs,
 		}
-		if autoscalingGroupVersion := (unversioned.GroupVersion{Group: "autoscaling", Version: "v1"}); registered.IsEnabledVersion(autoscalingGroupVersion) {
```

Comment:
It shouldn't be removed if we are going to serve autoscaling.Scale for /api/v1/replicationcontrollers. Reverting this change.